build_library/check_root

```diff
@@ -70,13 +70,36 @@ IGNORE_MISSING = {
                                  SonameAtom("x86_64", "libebt_vlan.so"),
                                  SonameAtom("x86_64", "libebtable_broute.so"),
                                  SonameAtom("x86_64", "libebtable_filter.so"),
-                                 SonameAtom("x86_64", "libebtable_nat.so")],
+                                 SonameAtom("x86_64", "libebtable_nat.so"),
+                                 SonameAtom("ppc_64", "libebtable_nat.so"),
+                                 SonameAtom("ppc_64", "libebt_802_3.so"),
+                                 SonameAtom("ppc_64", "libebt_among.so"),
+                                 SonameAtom("ppc_64", "libebt_arp.so"),
+                                 SonameAtom("ppc_64", "libebt_arpreply.so"),
+                                 SonameAtom("ppc_64", "libebt_ip.so"),
+                                 SonameAtom("ppc_64", "libebt_ip6.so"),
+                                 SonameAtom("ppc_64", "libebt_limit.so"),
+                                 SonameAtom("ppc_64", "libebt_log.so"),
+                                 SonameAtom("ppc_64", "libebt_mark.so"),
+                                 SonameAtom("ppc_64", "libebt_mark_m.so"),
+                                 SonameAtom("ppc_64", "libebt_nat.so"),
+                                 SonameAtom("ppc_64", "libebt_nflog.so"),
+                                 SonameAtom("ppc_64", "libebt_pkttype.so"),
+                                 SonameAtom("ppc_64", "libebt_redirect.so"),
+                                 SonameAtom("ppc_64", "libebt_standard.so"),
+                                 SonameAtom("ppc_64", "libebt_stp.so"),
+                                 SonameAtom("ppc_64", "libebt_ulog.so"),
+                                 SonameAtom("ppc_64", "libebt_vlan.so"),
+                                 SonameAtom("ppc_64", "libebtable_broute.so"),
+                                 SonameAtom("ppc_64", "libebtable_filter.so")],
 }
 
-USR_LINKS = ("/bin/", "/sbin/", "/lib/", "/lib32/", "/lib64/")
+#USR_LINKS = ("/bin/", "/sbin/", "/lib/", "/lib32/", "/lib64/")
+USR_LINKS = ("/sbin/", "/lib/", "/lib32/")
 
 IGNORE_SHEBANG = (
     b"*/python2.7/cgi.py",
+    b"*/python3.4/cgi.py",
```

Comment:
@glevand  @marineam Now we''re facing cross building issue with texinfo. See below build log.
https://bpaste.net/show/1da6fe86af91